src/main/java/moe/nightfall/vic/integratedcircuits/cp/legacy/LegacyLoader_1_Tunnels.java

```diff
@@ -0,0 +1,57 @@
+package moe.nightfall.vic.integratedcircuits.cp.legacy;
+
+import java.util.ArrayList;
+import java.util.BitSet;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import moe.nightfall.vic.integratedcircuits.cp.CircuitData;
+import moe.nightfall.vic.integratedcircuits.cp.legacy.LegacyLoader;
+import moe.nightfall.vic.integratedcircuits.misc.MiscUtils;
+import moe.nightfall.vic.integratedcircuits.misc.Vec2;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraftforge.common.util.ForgeDirection;
+
+public final class LegacyLoader_1_Tunnels extends LegacyLoader {
+	@Override
+	public int getVersion() {
+		return 1;
+	}
+
+	{
```

Comment:
This is only a minor thing, but I hate using the initialization blocks (looks a lot cleaner in scala), so rather make it a normal constructor.
Also, class names with `_` are definitely not my favorites